dashboard/app/controllers/api/v1/pd/facilitator_program_registrations_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::Pd::FacilitatorProgramRegistrationsController < ApplicationController
+  authorize_resource class: 'Pd::FacilitatorProgramRegistration', only: :create
```

Comment:
I'm confused; my understanding is that in `ability.rb` we define what it takes for someone to have the authority to create a registration, and here we say that in order to access this route you must fulfill those requirements. What are you suggesting I do?